setup.rst

```diff
@@ -401,6 +401,13 @@ and ``make``::
 
     $ make -s -j2
 
+.. note:: 
```

Comment:
There's a trailing space here.
I'd recommend to configure your editor to remove them automatically.

Reply:
Thanks very much for the editor advice @ezio-melotti. I was updating the doc directly online on github but I will get to using an editor soon enough :) I will wait to hear back from @tiran as I will certainly remove completely the note and update the brew line command.

Reply:
From GitHub it's more difficult to catch trailing whitespace -- I sometimes do this mistake myself.  I started to check that manually before submitting the PR by highlighting the text.  Even if you still miss one, the checks on the PR will tell you and you can fix the error in another commit.